apps/questapi/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "@tsconfig/node18/tsconfig.json",
```

Comment:
we might want to change towards `tsconfig/node22` since `Node v 22.11.0` is released and labelled as `LTS` according to [this and will be maintained until April 2027](https://nodejs.org/en/blog/release/v22.11.0).